packages/tools/addon/components/cs-composition-panel-header.js

```diff
@@ -1,9 +1,18 @@
 import Component from '@ember/component';
+import { inject as service } from '@ember/service';
+import { task } from 'ember-concurrency';
 import layout from '../templates/components/cs-composition-panel-header';
 
 export default Component.extend({
   layout,
+  cardstackData: service(),
   tagName: 'header',
   classNames: ['cs-composition-panel-header'],
-  classNameBindings: ['editingEnabled:enabled:disabled']
+  classNameBindings: ['editingEnabled:enabled:disabled'],
+
+  getModelTitle: task(function * () {
```

Comment:
I'm not sure we need to launch a task on init here, we could just use a CP:

```js
title: computed('model', function() {
  return this.cardstackData.getCardMeta(this.model, 'title');
})
```
(that's simpler and should `model` ever change, it will be recomputed)

Reply:
Sounds good, thanks!